.circleci/circleci_cli.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# This script is convenient to run circlici job on lical macine.
+# What this script does:
+# - converts config to 2.0 version because CircleCi CLI does not support config version higher than 2.0.
+# - set root user for all jobs with 'setup_remote_docker',
+#   because 'circleci' does not have permisions to run docker commands.
```

Comment:
Which user does the command run under by default?

One option would also be to fail and print out an error message and say that that user needs to be added to local ``docker`` or similar.

Reply:
I am afraid it does not allow us to add a user or to add user to a group without root permissions too. Also if we use `sudo`, it works only locally.